getJigsawJDK.sh

```diff
@@ -11,9 +11,9 @@ JDK_DESTINATION=$(echo ```pwd```)
 JDK_FOLDER_NAME="jdk-9"
 JDK_HOME_OS_SPECIFIC="$JDK_DESTINATION/$JDK_FOLDER_NAME"
 
-JDK_DOWNLOAD_HOME_URL="https://jdk9.java.net"
-BUILD_NUMBER=$(curl $JDK_DOWNLOAD_HOME_URL/download/ | grep build | awk '{print $9}' | tr -d "</a><br")
-JDK_DOWNLOAD_BASE_URL="http://www.java.net/"
+JDK_DOWNLOAD_HOME_PAGE="http://jdk.java.net/9/"
+BUILD_NUMBER=$(curl $JDK_DOWNLOAD_HOME_PAGE/ | grep "Most recent build" | awk '{print $4}' |  tr -d "</h2>" | grep -oP "jdk\-9\+\K\w+")
```

Comment:
Many, could the reason be that this is not the jigsaw build? I just learned of this, http://jdk.java.net/jigsaw, which contains the OS X .tar.gz build.